client/my-sites/plans-features-main/index.jsx

```diff
@@ -91,13 +97,24 @@ class PlansFeaturesMain extends Component {
 			);
 		}
 
-		const plans = filter(
-			[
-				hideFreePlan ? null : PLAN_FREE,
-				isPersonalPlanEnabled ? PLAN_PERSONAL : null,
+		let signupPlans = [
+			hideFreePlan ? null : PLAN_FREE,
+			isPersonalPlanEnabled ? PLAN_PERSONAL : null,
+			PLAN_PREMIUM,
+			PLAN_BUSINESS
+		];
+
+		if ( abtest( 'signupPlansReorderTest' ) === 'modified' ) {
+			signupPlans = [
+				PLAN_BUSINESS,
 				PLAN_PREMIUM,
-				PLAN_BUSINESS
-			],
+				isPersonalPlanEnabled ? PLAN_PERSONAL : null,
+				hideFreePlan ? null : PLAN_FREE,
+			];
+		}
+
+		const plans = filter(
+			signupPlans,
 			value => !! value
```

Comment:
Thanks for the suggestion. That's definitely more succinct. I'll give it a try and deploy if it works well otherwise I'll go with the existing code :)